### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by XiaoqZhang**
1. Redirect the Install Prerequisites to a new link. 
2. Create a conda environment with higher python version and load required packages automatically.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ conda init
 ```
 Create an enviroment `aiida1` with python 3.6 and activate this new environment
```
Comment (ltalirz) [inline README.md]: would you mind updating the python version here as well?

File: README.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ conda init
...
 ```
-conda create -n aiida1 python=3.6 -y
```
Comment (ltalirz) [inline README.md:19]: Have you tested this?
I think one usually first does the `conda create` command, and then the `conda install -c conda-forge ...`

**Review by ltalirz — CHANGES_REQUESTED** [review]
thanks a lot for the pull request @xiaoqiepfl !

Just two small comments from my side

File: README.md (hunk 2 above)
Comment (XiaoqZhang) [reply README.md:19]: Thanks for @ltalirz. Yes, that would be better.

**Comment by ltalirz** [comment]
thanks @xiaoqiepfl - just as a final thing: if you look inside the `install_editable.sh` file you will notice that it installs aiida-core (+ plugins) in editable mode.

This is somewhat on purpose since it makes it easy to make changes in `aiida-core` and push them back upstream.
When doing this, installing the `aiida-core` package through conda isn't required (and the `aiida-core.services` should be comvered by the "prerequisites" section).

I.e. unless you remove the added line again (unless you think there is a good reason to keep it).

**Comment by ltalirz** [comment]
thanks a lot @xiaoqiepfl !

**Pull request merged**